ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+### Context
+
+* [ ] **mapbox-gl-js version**: 
```

Comment:
No, I don't think we can ask that of all users. For any deployable software, a lot of people will be using an out-of-date version and won't be able to immediately update.